sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/FeedbackModel.kt

```diff
@@ -3,6 +3,7 @@ package co.touchlab.sessionize
 import co.touchlab.sessionize.api.FeedbackApi
 import co.touchlab.sessionize.api.NetworkRepo
 import co.touchlab.sessionize.db.SessionizeDbHelper
+import co.touchlab.sessionize.platform.NotificationsModel.cancelFeedbackNotifications
```

Comment:
I'm getting a compile error on this line. It looks like cancelFeedbackNotifications was deleted from NotificationsModel.